interview-questions.md

```diff
@@ -9,59 +9,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The `this` command is used to call methods that are in the same string. Instead of calling to the class and method, it allows coders to shortcut the code.
 
-  Researched answer:
+  Researched answer: `this` refers to an object and depending on what type of objects it is, changes how `this` keyword is called.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a application created by Facebook to handle a large amount of users and their user interface. The reason you would use it is it's ability to quickly refresh a page when new inputs are added by other users.
 
-  Researched answer:
+  Researched answer: React is a JavaScript library which is used for user interfaces on web pages or mobile apps.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: The constructor is the lifecycle method that is required. It allows the parents classes to pass data to the child classes without having to make the same information in the child classes.
 
-  Researched answer:
+  Researched answer: React has three lifecyle methods. Mounting, updating, and unmounting which help show and update what is shown on the web page through the DOM.
```

Comment:
The render() method is the only required lifecycle method in React.  React's whole purpose is to display content to a user and the render() method is the method that displays our content.  Because only one element is displayed within the render() method, we have to wrap everything inside it with React fragments.